tests/Algorithms/HungarianAlgorithmTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Bearded.Utilities.Algorithms;
+using FsCheck.Xunit;
+using Xunit;
+
+namespace Bearded.Utilities.Tests.Algorithms
+{
+    public class HungarianAlgorithmTests
+    {
+        [Fact]
+        public void Run_EmptyCostMatrix_DoesntFail()
+        {
+            Assert.Empty(HungarianAlgorithm.Run(new float[0, 0]));
+        }
+
+        [Property]
+        public void Run_OneSource_AssignsTarget(float cost)
```

Comment:
This test also tests two things. Please use custom generators (or parameterise the existing one if possible) and split into two tests.